### DISCUSSION TITLE
where do we submit issues

### DISCUSSION THREAD

**Discussion by drdamour** [mulesoft-anypoint/terraform-provider-anypoint#3] [General]
the relationship between this repo and it's upstream https://github.com/mulesoft-consulting/terraform-provider-anypoint isn't very clear. where should we submit issues...this repo doesn't even allow for issues just discussions.

**Comment by drdamour** [comment]
trying to run this answer to ground so shameless tagging some commiters @soufi  @souf @rubycube

**Reply by soufi** [reply]
Hey @drdamour, 

We have created this repository because we couldn't get the required level of permissions from mulesoft-consulting. 
You can open an issue on the mulesoft-consulting. But I'm the main maintainer and I no longer work for Mulesoft. 
Therefore I would suggest to get in touch with me directly on discord [server](https://discord.gg/pSx9vaZ3) or here. 
I just saw your PR, please tag me when you need validation, because I don't get any notifications otherwise. 

Thank you for your patience @drdamour

**Reply by drdamour** [reply]
@soufi not fully understanding...if we need a bug fix, will it be fixed here and published or fixed here, then merged into the upstream muelsoft-consulting and then published from there?

**Reply by soufi** [reply]
@drdamour  As of now, if you need bug fix, it will be fixed here and published from here since the terraform registry is linked to this repository.  a PR will be sent to the mulesoft-consulting repo.

**Reply by drdamour** [reply]
@soufi then i suggest allowing issues to be opened in this repo, seems like the upstream is mostly irrelevant at this point

**Reply by soufi** [reply]
@drdamour I was hoping that people from the editor would take some of their time to collaborate since this project has a great success ... but so far I'm the only one carrying so far. 

I have enabled issues here. 

thank you for your interest @drdamour.
